src/pages/user-dashboard/user-dashboard.ts

```diff
@@ -167,6 +144,16 @@ export class UserDashboardPage {
 			this.assessments = data.assessments;
 			this.assessments = JSON.parse(JSON.stringify(this.assessments));
 			console.log(this.assessments);
+			for (var assessment of this.assessments) {
+				var answeredQuestions = assessment.questions.filter((q: any) => q.answers.length > 0 && q.mrLevel == assessment.targetMRL);
+				var lengthOfAssessment = assessment.questions.filter((q:any) => q.mrLevel == assessment.targetMRL);
+				var answered = answeredQuestions.length;
+				var possible = lengthOfAssessment.length;
```

Comment:
Replace the var's with let for 
```
var answered
var possible
``` 
It'll make for safer code to limit the scope of answered and possible to within the function.

And it doesn't seem like they need to be used outside?